src/html-duration-picker.js

```diff
@@ -124,7 +124,7 @@ export default (function() {
   const validateInput = (event) => {
     const sectioned = event.target.value.split(':');
     if (sectioned.length !== 3) {
-      event.target.value = '00:00:00'; // fallback to default
+      event.target.value = event.target.dataset.duration || '00:00:00'; // fallback to default
```

Comment:
@PeterMorganGH This is working like a charm! 
However, we may need to validate the input first, before using it. I tested this with the following scenario:
` data-duration="00:2x'&#x3C;script&#x3E;alert()&#x3C;/script&#x3E;5:00"`. 
The good news is it does not break the page, but the starting value of the picker is not a valid duration value. Needless to say each time I each time an invalid  duration entry is made, it falls back to another invalid duration value.

How do you propose we go around this?

Reply:
You bring up a good point. I think a Regular Expression could be used to check if the data property is in the valid format. If it is, it gets assigned. If the format is incorrect, it defaults to `00:00:00`. Let me see if I can figure this out.

Reply:
Thank you @PeterMorganGH you've really helped us out here

Reply:
I pushed a commit with a regular expression check that specifically looks for a string formatted as `##:##:##`.

Reply:
@PeterMorganGH that is much better. This is amazing work. I'm considering refactoring the function ```validateInput``` to use a RegExp too. Thank you tons